packages/react-native/React/Base/RCTConvert.m

```diff
@@ -1538,13 +1637,13 @@ + (NSString*)accessibilityRoleFromTrait:(NSString*)trait
   return role;
 }
 
-+ (NSString *)accessibilityRoleFromTraits:(id)json
++ (NSString *)accessibilityRoleFromTraits:(id)json withAriaMappings:(BOOL) useAriaMappings
```

Comment:
Nit:  I think the more objective-c ish way to write this would be `usingAriaMappings:(BOOK) useAriaMappings`. "withAriaMappings" feels like an object is about to be passed in to me

Reply:
But also our official guidelines are on `aka.ms/objc-guide`
https://microsoft.github.io/objc-guide/MethodsAndImplementations/Naming.html

Reply:
Thanks!